config/kubernetes/org.yaml

```diff
@@ -1574,7 +1480,7 @@ teams:
     - timothysc
     privacy: closed
   kubernetes-blog-maintainers:
-    description: Maintaining the Kubernetes blog on kubernetes.io
+    description: Write access on kubernetes.io
```

Comment:
I think this doesn't describe what the team is actually responsible for.
`kubernetes-blog-maintainers` is assigned read access to k/website and is represented in [the blog OWNERS file](https://github.com/kubernetes/website/blob/master/content/en/blog/OWNERS). 

I don't know about `kubernetes-blog`, but it looks like that one is mentioned as team [here](https://github.com/kubernetes/community/blob/b6dcdba633c2709cfd86fe39844e360163eed6f8/sigs.yaml#L1152).

Maybe it is best to keep the description for now, or describe it as `Maintainers of the Kubernetes blog` and then when @zacharysarah is back from his holiday discuss whether we can consolidate the teams.

Reply:
That make sense, I will update it.